tls/s2n_crl.c

```diff
@@ -44,6 +44,7 @@ int s2n_crl_load_pem(struct s2n_crl *crl, uint8_t *pem, size_t len) {
 
     uint32_t data_size = s2n_stuffer_data_available(&der_out_stuffer);
     const uint8_t *data = s2n_stuffer_raw_read(&der_out_stuffer, data_size);
+    POSIX_ENSURE_REF(data);
```

Comment:
Nice catch!

Reply:
We have simple-mistakes to thank for this one...

Reply:
The API really should have been

```c
const uint8_t *data = NULL;
POSIX_GUARD(s2n_stuffer_raw_read(&der_out_stuffer, data_size, &data));
```